build.gradle.kts

```diff
@@ -1,10 +1,11 @@
+
 plugins {
-    java
-    id("jacoco")
-    id("org.springframework.boot") version "2.7.12"
-    id("io.spring.dependency-management") version "1.0.15.RELEASE"
-    id("org.jsonschema2dataclass") version "4.2.0"
-    id("info.solidsoft.pitest") version "1.9.11"
+	java
+	id("jacoco")
+	id("org.springframework.boot") version "2.7.12"
```

Comment:
@patelila I think your editor must be changing spaces into tabs. Two concerns here:
1. This PR contains whitespace changes, which are not part of the ticket goal, so distract. Please ensure you read through your PR diff and remove anything that is not a conscious change for the better.
2. Please can you establish what is normal practice in Justice Digital's Java community - tabs or spaces. Set your editor accordingly. And if the files are wrong then change them all at once in a separate PR.

Reply:
@davidread CMA uses a github action for code formatting -       
- uses: ministryofjustice/github-actions/code-formatter@main

Reply:
@starswan Nice idea. [That one](https://github.com/ministryofjustice/github-actions/blob/main/code-formatter/README.md) doesn't do the java or groovy files, unfortunately, but there's probably another out there that does

Reply:
weird - something is formatting my Java code for me during my PRs